Refactor user profile file uploads to separate endpoints

NKSurprise opened the pull request:
Moved CV and profile picture uploads from UpdateUserProfileRequest to dedicated endpoints and request models. Updated controller, service interfaces, and implementations accordingly. Validation for file uploads in UpdateUserProfileRequestValidator is now commented out, reflecting the new separation of concerns.

NKSurprise committed:
Refactor user profile file uploads to separate endpoints

Moved CV and profile picture uploads from UpdateUserProfileRequest to dedicated endpoints and request models. Updated controller, service interfaces, and implementations accordingly. Validation for file uploads in UpdateUserProfileRequestValidator is now commented out, reflecting the new separation of concerns.

ImSk1 commented on src/SkillBridge/Infrastructure/Validation/Validators/UpdateUserProfileRequestValidator.cs:
Remove unnecessary comments.

ImSk1 commented on src/SkillBridge/Services/UserProfile/UserProfileService.cs:
```diff
@@ -122,10 +122,85 @@ public async Task<UserProfileResponse> UpdateAsync(UpdateUserProfileRequest requ
+            _dbContext.UserProfiles.Update(userProfile);
+            await _dbContext.SaveChangesAsync();
+
+            var username = (await _managementApiClient.Users.GetAsync(userProfile.Id)).UserName;
```
Remove call to Auth0 API.

NKSurprise committed:
Remove CV and profile picture handling from user profile

Commented-out code for CV and profile picture upload and validation has been removed from both the validator and service. Logging now references profile IDs instead of usernames, simplifying the update and retrieval processes.
